### ISSUE TITLE
Dataset

### ISSUE THREAD

**Issue by hqlin2018** [stamdlee/DeepLearningFramework#2]
i can not see any dataset in there ,could you tell how to get the dataset?

**Comment by aodongli** [comment]
I would recommend contacting Huawei for their dataset. Or you can use some public dataset instead.

**Comment by hqlin2018** [comment]
> I would recommend contacting Huawei for their dataset. Or you can use some public dataset instead.

thanks for you advise, i have got the dataset from Huawei, the dataset have two package as repos.tar.gz and train.tar.gz .but now i don't know how use the dataset in the code. when i want to use the dataset , what i need to do process the dataset first?

**Comment by aodongli** [comment]
Yes. You need to preprocess the data. Sorry that I didn't upload the preprocessing code. If I remember correctly, I used similar preprocessing with 
https://www.tensorflow.org/tutorials/text/nmt_with_attention
which is using batches and adding `MASK` to make batch to be the same length.

In fact, I believe almost all NLP systems use similar preprocessing.

**Comment by scoyer** [comment]
I want to ask a question. Have you trained the vanilla seq-to-seq model with the whole dataset which contains 4000k sentences. I tried it and found it was difficult to converge when I use the vanilla seq-to-seq model.

**Comment by hqlin2018** [comment]
> Yes. You need to preprocess the data. Sorry that I didn't upload the preprocessing code. If I remember correctly, I used similar preprocessing with
> https://www.tensorflow.org/tutorials/text/nmt_with_attention
> which is using batches and adding `MASK` to make batch to be the same length.
> 
> In fact, I believe almost all NLP systems use similar preprocessing.

Ok, thank you very much, i woudl try it again.

**Comment by hqlin2018** [comment]
> I want to ask a question. Have you trained the vanilla seq-to-seq model with the whole dataset which contains 4000k sentences. I tried it and found it was difficult to converge when I use the vanilla seq-to-seq model.

Sorry , i have no idea about vanilla seq2seq model.

**Comment by scoyer** [comment]
> > I want to ask a question. Have you trained the vanilla seq-to-seq model with the whole dataset which contains 4000k sentences. I tried it and found it was difficult to converge when I use the vanilla seq-to-seq model.
> 
> Sorry , i have no idea about vanilla seq2seq model.

What about the model in this repository.

**Comment by hqlin2018** [comment]
sorry, i didn't have a try&nbsp;



发自我的iPhone


------------------ Original ------------------

**Comment by lsx1995** [comment]
> i can not see any dataset in there ,could you tell how to get the dataset?

Could you sent me the dataset? I really need it for my research.Thank you so much!!!

**Comment by hqlin2018** [comment]
are you a student ？



发自我的iPhone


------------------ Original ------------------

**Comment by lsx1995** [comment]
yep,i'm a student from&nbsp;China University of Geosciences




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"hqlin2018"<notifications@github.com&gt;;
发送时间:&nbsp;2019年12月30日(星期一) 晚上9:39
收件人:&nbsp;"aodongli/DeepLearningFramework"<DeepLearningFramework@noreply.github.com&gt;;
抄送:&nbsp;"黎尚雄"<554373559@qq.com&gt;;"Comment"<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [aodongli/DeepLearningFramework] Dataset (#2)



are you a student ？ 
 
 
 
 发自我的iPhone 
 
 
 ------------------ Original ------------------
